2023-RADIUSS-AWS/JupyterNotebook/aws/eksctl-autoscale-config.yaml

```diff
@@ -0,0 +1,76 @@
+# https://www.arhea.net/posts/2020-06-18-jupyterhub-amazon-eks
```

Comment:
Oh wow we needed an entire new config? Sheesh. Just so I understand:

1. This config is for when I want to customize the storage class
2. The eksctl-config.yaml is for when I don't

But both are for autoscaling right?

Reply:
Could you add a note in the README where we create the cluster with eksctl examples for creating both, and then the context under which you'd want to do that?